e2e/test/scenarios/onboarding/embedding-homepage.cy.spec.ts

```diff
@@ -0,0 +1,78 @@
+import {
```

Comment:
I created a new file instead of adding to the setup one, this doesn't do the whole setup so they're much faster and we can test more scenarios